aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+      }
+
+Here is a more robust example, adding ``<remarks />`` where the content can be just text or adding the JSON or xml object for further documentation of the method.
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Controllers/TodoController.cs
+    :language: c#
+    :start-after: Create_Method
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 4-14
+
+Notice the enhancement of the UI with these additional comments.
+
+.. image:: web-api-help-pages-using-swagger/_static/xml-comments-extended.png
+
+
+DataAnnotations
+'''''''''''''''
+Another supported feature is using the ``System.ComponentModel.DataAnnotations`` to decorate the API controller, methods and classes to help drive the Swagger components.
+
+For example, adding the ``[Required]`` annotation to the ``Name`` property of the TodoItem class will change the ModelSchema information in the UI. Other DataAnnotations such as ``[Produces("application/json"]``, RegularExpression validators and more will further detail the information delivered in the generated page.  What is important is that the more information that is documented in the code, the more "auto" generated information is pushed to the consumer.
+
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Models/TodoItem.cs
+  :language: c#
+  :emphasize-lines: 10
+
+
+Describing Response Types
+'''''''''''''''''''''''''
+Consuming developers are probably most concerned with what is returned; specifically response types, error codes (if not standard). These are handled in the xml comments and DataAnnotations.
+
+Take the ``Create()`` method for example, currently it returns only "201 Created" response by default. That is of course if the item is in fact created, or a "204 No Content" if no data is passed in the POST Body.  However, there is no documentation to know that or any other response. That can be fixed by adding the following piece of code.
+
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Controllers/TodoController.cs
+    :language: c#
+    :start-after: Create_Method
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 17,18,20,21
+
+.. image:: web-api-help-pages-using-swagger/_static/data-annotations-response-types.png
+
+Customizing the UI
+''''''''''''''''''
+The stock UI is very functional as well as presentable, however when building documentation pages for your API you want it to represent your brand or look and feel. 
+
+Accomplishing that task with the Swashbuckle components is simple but requires adding the resources to serve static files that would not normally be included in a Web API project and then building the folder structure to host those files.
+
+Add the ``"Microsoft.AspNetCore.StaticFiles": "1.0.0-*"`` NuGet package to the project either using one of the methods mentioned prior in this tutorial and then open the *Startup.cs* file to add the necessary code for the static file middleware.
```

Comment:
Add ``"Microsoft.AspNetCore.StaticFiles": "1.0.0"``  to the project. 
By now they should know how, no need to tell them ..by one of the ..
new paragraph
Enable static files:
// show code here, OK to use .. code-block:: c#